jboss-kie-workbench/added/launch/jboss-kie-workbench.sh

```diff
@@ -53,6 +53,7 @@ function configure() {
     configure_server_access
     configure_openshift_enhancement
     configure_workbench_profile
+    check_SSH_Files
```

Comment:
@desmax74 Not a big deal, but all the other functions are all-lowercase, so it would probably be good to keep the same convention, and rename this function to check_ssh_files.

Reply:
fixed